README.md

```diff
@@ -19,25 +19,70 @@ yarn add --dev @netlify/ipx
 Create `netlify/functions/ipx.ts`:
 
 ```ts
-import { createIPXHandler } from '@netlify/ipx'
+import { createIPXHandler } from "@netlify/ipx";
 
 export const handler = createIPXHandler({
-  domains: ['images.unsplash.com']
-})
+  domains: ["images.unsplash.com"],
+});
 ```
 
 Now you can use IPX to optimize both local and remote assets ✨
 
 Resize `/test.jpg` (in `dist`):
 
 ```html
-<img src="/.netlify/functions/ipx/w_200/static/test.jpg"/>
+<img src="/.netlify/functions/ipx/w_200/static/test.jpg" />
 ```
 
 Resize and change format for a remote url:
 
 ```html
-<img src="/.netlify/functions/ipx/f_webp,w_450/https://images.unsplash.com/photo-1514888286974-6c03e2ca1dba"/>
+<img
+  src="/.netlify/functions/ipx/f_webp,w_450/https://images.unsplash.com/photo-1514888286974-6c03e2ca1dba"
+/>
+```
+
+## Remote Patterns
+
+Instead of setting an allowlist on `domains`, you may wish to use the option `remotePatterns`. This method allows wildcards for hostname as well as pathname segments.
+
+`remotePatterns` is an array that contains RemotePattern objects:
+
+```ts
```

Comment:
This repo will help us implement Next's remotePatterns canary, but isn't strictly tied to it (not importing any next packages).

Reply:
Gotcha. I think I thought it was Next.js specific because of this.

```typescript
// NextJS RemotePattern
export interface RemotePattern {
  protocol?: 'http' | 'https';
  hostname: string;
  port?: string;
  pathname?: string;
}
```

Since it's not Next.js specific, consider removing the comment above the interface.